### PR TITLE
Added in examples and link to tutorial for imaging.rst.

### PR DESCRIPTION
**Pull request by gsenesac**
Added an example section to the imaging.rst file as well as a link to a Pillow tutorial.

### CODE REVIEW

File: docs/scenarios/imaging.rst [hunk 1]
```diff
@@ -28,3 +28,32 @@ After that, it's straightforward:
 .. code-block:: console
 
     $ pip install Pillow
+
+Example
+~~~~~~~~~~~~
```
Comment (sigmavirus24) [inline docs/scenarios/imaging.rst]: These are supposed to match the length of the title, e.g.,

```restructuredtext
Example
~~~~~~~
```

**Comment by sigmavirus24** [comment]
Ping me when this has been updated @gsenesac

**Comment by gsenesac** [comment]
Fixed.

**Comment by sigmavirus24** [comment]
Thanks @gsenesac

**Pull request merged**